lib/driver.js

```diff
@@ -711,6 +660,56 @@ class XCUITestDriver extends BaseDriver {
     }
   }
 
+  /**
+   * Start the simulator and initialize based on capabilities
+   */
+  async initSimulator() {
+    // @ts-expect-error - do not assign arbitrary properties to `this.opts`
+    const device = this.opts.device;
+
+    if (this.opts.shutdownOtherSimulators) {
+      this.ensureFeatureEnabled(SHUTDOWN_OTHER_FEAT_NAME);
+      await shutdownOtherSimulators(device);
+    }
+
+    await this.startSim();
+
+    if (this.opts.customSSLCert) {
+      // Simulator must be booted in order to call this helper
+      // @ts-expect-error - do not assign arbitrary properties to `this.opts`
+      await this.opts.device.addCertificate(this.opts.customSSLCert);
```

Comment:
this.opts.device -> device

Reply:
oops, fixed https://github.com/appium/appium-xcuitest-driver/pull/1972/commits/2e6a3c4e87d1f3787af67e4d0bcdfcfd4f178c0b

thank you for reviewing